Lesson_11/l11_h01_generators.py

```diff
@@ -0,0 +1,34 @@
+"""
+HW:11[Volkov Anton].
+
+Write a generator function that takes a range value and chunk_size as input
+and produces output as lists with length equal to the chunk size.
+Example:
+Call: chunk_generator(range(11), chunk_size=3)
+Output:
+[0, 1, 2]
+[3, 4, 5]
+[6, 7, 8]
+[9, 10]
+"""
+
+
+def chunk_generator(range_value, chunk_size):
```

Comment:
Please pay attention that according task description we should send range not as int but as range value
`chunk_generator(range(11), chunk_size=3)`